tealium_angular.js

```diff
@@ -57,7 +57,10 @@ angular.module("TealiumHelper", ["TealiumHelper.data"])
     addViewIdMapEntry: tealiumDataProvider.addViewIdMapEntry,
     $get: [ "tealiumData", "$location", function(tealiumData, $location) {
       if (!config.account || !config.profile) {
```

Comment:
This is the main condition which was there before – it will protect the module from initialising if the config isn't supplied.

Reply:
I see (sort of).. can we change your 'if' to the following instead?

if (window.console) { 
  console.warn("Tealium configuration is not valid. Please set both account and profile."); 
}

Reply:
Yep @tygavin that's done.